### ISSUE TITLE
Add man_made=threshing_floor

### ISSUE THREAD

**Issue by AntMadeira** [openstreetmap/id-tagging-schema#310]
I would like to request the inclusion of man_made=threshing_floor in iD's presets.
https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dthreshing_floor

It would be nice to add the surface=* field also.

**Comment by matkoniecz** [comment]
https://taginfo.openstreetmap.org/tags/man_made=threshing_floor

<img width="346" height="294" alt="Image" src="https://github.com/user-attachments/assets/9472ae44-6d4d-4948-af24-c5292f82eec4" />

looks like over half was added by problematic import (EDIT: it was not import per below)

**Comment by matkoniecz** [comment]
makes me more dubious should it be supported at the current stage

**Comment by AntMadeira** [comment]
It was not an import. Those were mapped based on aerial imagery in a span of a few days.

> looks like over half was added by problematic import

**Comment by matkoniecz** [comment]
sorry!

then by a single edit wave - and still, in large part later deleted

**Comment by AntMadeira** [comment]
If I recall correctly, there was a community effort to fix many mistagged threshing floors and this was also done in a short amount of time.
Anyway, it seems to have a steady increase of a thousand elements over these last years.

**Comment by matkoniecz** [comment]
Yes, I am still trying to judge - also given its relative obscurity.

At least it is unlikely to be mistakenly used, I think?
